docs/getting-started-guides/aws.md

```diff
@@ -29,10 +29,10 @@ As a minimum you will need to set these environment variables -
 
 ```
 APOLLO_PROVIDER=aws
-AWS_ACCESS_KEY_ID
-AWS_SECRET_ACCESS_KEY
-AWS_SSH_KEY
-AWS_SSH_KEY_NAME
+TF_VAR_aws_access_key
+TF_VAR_secret_key
+TF_VAR_key_file
+TF_VAR_key_file
 ATLAS_TOKEN
```

Comment:
ATLAS TOKEN is a special case as it is being using at the moment by terraform and ansible